packages/walt-compiler/src/tokenizer/__tests__/tokenizer-spec.js

```diff
@@ -48,6 +53,47 @@ test("ignores comments", t => {
   t.snapshot(result);
 });
 
+test("ignores multiline comments", t => {
+  const stream = new Stream(`
+    /* multiline
+     *  comment
+     * */
+    2 * 2`);
+  const tokenizer = new Tokenizer(stream);
+  t.snapshot(tokenizer.parse());
+});
+
+test("ignores one-liner multiline comments", t => {
+  const stream = new Stream(`
+    /* comment  */
+    2`);
+  const tokenizer = new Tokenizer(stream);
+  t.snapshot(tokenizer.parse());
+});
+
+test("ignores singleline tokens within multiline and vice-versa", t => {
+  const stream = new Stream(`
+    /* // multiline comment /*
+    // /* */ 4 single line
```

Comment:
I like the idea of this spec, but isn't this just one large multi-line comment?
 
line 76 ends in `/*` which I don't think closes a multi-line comment